build_project/post_build.go

```diff
@@ -0,0 +1,40 @@
+package build
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tarantool/cartridge-cli/common"
+	"github.com/tarantool/cartridge-cli/project"
+)
+
+func PostRun(projectCtx *project.ProjectCtx) error {
+	// post-build
+	postBuildHookPath := filepath.Join(projectCtx.BuildDir, postBuildHookName)
+
+	if _, err := os.Stat(postBuildHookPath); !os.IsNotExist(err) {
```

Comment:
But it could be another error type. Permission denied or smth else.

I think we should run hook only if stat doesn't return error.